src/edu/psu/sweng500/team8/gui/SudokuGUI.java

```diff
@@ -606,11 +653,11 @@ public String getDescription() {
 	 * SAVE Action
 	 * 
 	 * @param evt
-	 * @throws FileNotFoundException
+	 * @throws NotFoundException
 	 */
-	private void btnSaveActionPerformed(ActionEvent evt)
+	private void btnSaveActionPerformed()
 			throws FileNotFoundException {
-
+		this.gameChanged = false;
```

Comment:
This is a potential bug if the user cancels the file chooser. Move this to after the file choose dialog